### PR TITLE
Bugfix/scrollable brick list

### LINKED ISSUE #3: [Bug]: Grouped Area Brick List not scrollable #12005
https://github.com/pimcore/pimcore/issues/12005

### PR DESCRIPTION
**Pull request by cameronfromtorq**
resolves #3 
will need this testbed branch for bricks: https://github.com/TorqIT/pimcore-testbed/tree/testing/some-data-and-bricks

image

resize window...
image

### CODE REVIEW

File: bundles/AdminBundle/Resources/public/js/pimcore/document/editables/areablock.js [hunk 1]
```diff
@@ -1056,6 +1057,22 @@ pimcore.document.editables.areablock = Class.create(pimcore.document.area_abstra
...
+                var winH = 0;
+                if (document.body && document.body.offsetHeight) {
+                    winH = document.body.offsetHeight;
+                }
+                if (document.compatMode=='CSS1Compat' && document.documentElement && document.documentElement.offsetHeight ) {
+                    winH = document.documentElement.offsetHeight;
+                }
+                if (window.innerHeight) {
+                    winH = window.innerHeight;
+                }
```
Comment (lukemacausland) [inline bundles/AdminBundle/Resources/public/js/pimcore/document/editables/areablock.js]: Should these be if / else if / else if?

Comment (cameronfromtorq) [reply bundles/AdminBundle/Resources/public/js/pimcore/document/editables/areablock.js]: I could probably write it that way but Its meant to determine what API the current browser is using to provide document height and widths.

**Comment by lukemacausland** [comment]
@cameronfromtorq, would this help? https://docs.sencha.com/extjs/6.2.0/classic/Ext.window.Window.html#cfg-monitorResize

**Comment by lukemacausland** [comment]
@cameronfromtorq I would like to maybe setup a quick 15-30 minute meeting to go over this and investigate a bit further. I think I may have found something that will work just using css. I will test it out on my side when I get my local environment setup.

Maybe set max-height to a percentage (80%)? Than find the `pimcore_areablock_toolbar` class and add an `overflow: auto` to it?

**Comment by lukemacausland** [comment]
image

**Review by lukemacausland — APPROVED** [review]
@cameronfromtorq want to take a quick look at this

**Review by evanjamesjackson — APPROVED** [review]
Great

**Pull request closed without merging**